LayoutTests/fast/css/getComputedStyle/grid-template-node-not-connected-crash.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<style>
+#child {
+  grid-template: 10px / 10px;
+}
+</style>
+<div>
+  <div id="child"></div>
+</div>
+<div id="target">This test passes if it doesn't crash</div>
+<script>
+if (window.testRunner)
+    testRunner.dumpAsText();
+let styleDeclaration = window.getComputedStyle(document.getElementById("child"));
+document.querySelector("body > div:nth-child(1)").textContent = "";
+let gridTemplate = styleDeclaration.gridTemplate;
```

Comment:
This test should check the value (null or empty string) rather than just checking for a crash. Also I'd like to see this kind of test in WPT since we want matching behavior in all browsers for interoperability. Don't want the error case to behave differently in other browsers.

Reply:
Changed the test and created a WPT export 
https://github.com/web-platform-tests/wpt/pull/41946